rep-2000.rst

```diff
@@ -306,6 +307,111 @@ Build System Support:
++==============+=======================+==============================+========================+========================+
+|    amd64     |        Tier 1         |           Tier 2*            |          Tier 1*       |        Tier 3          |
++--------------+-----------------------+------------------------------+------------------------+------------------------+
+|    arm64     |        Tier 1         |                              |                        |        Tier 3          |
++--------------+-----------------------+------------------------------+------------------------+------------------------+
+|    arm32     |        Tier 2         |                              |                        |        Tier 3          |
++--------------+-----------------------+------------------------------+------------------------+------------------------+
+
+
+" * " Individual binary packages are not currently available for these platforms.
+Binary releases are provided as a single archive per platform containing all packages in [this repos file](https://github.com/ros2/ros2/blob/master/ros2.repos).
+
+" [s] " Compilation from source, the ROS buildfarm will not produce any binary packages for these platforms.
+
+Middleware Implementation Support:
+
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| Middleware Library       | Middleware Provider | Support Level | Platforms                   | Architectures                        |
++==========================+=====================+===============+=============================+======================================+
+|  rmw_fastrtps_cpp        | eProsima Fast-RTPS  | Tier 1        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+|  rmw_connext_cpp         | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64/arm32 |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_opensplice_cpp       | ADLink OpenSplice   | Tier 2        | All Platforms except Debian | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-RTPS  | Tier 2        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+|  rmw_connext_dynamic_cpp | RTI Connext         | Tier 2        | All platforms except Debian | All architectures except arm64/arm32 |
```

Comment:
We should remove this since we haven't been building this RMW impl. for a long time. It has an IGNORE marker file in the repo and not even an option in CI to build it.